experimental/Schemes/src/Resolution_structure.jl

```diff
@@ -462,6 +604,31 @@ end
 ##################################################################################################
 # desingularization workers
 ##################################################################################################
+function principalization(I::AbsIdealSheaf; algorithm::Symbol=:BEV)
```

Comment:
could you please add a test for this function?

Reply:
But this can also be done later.

Reply:
This function is one of the empty hulls -- to be filled with the next PR (algorithm: CJS). As I already wrote above, the low coverage is due to some not yet used analogues for the embedded case and some still empty hulls.